src/main/resources/application.yml

```
@@ -39,3 +39,17 @@ jwt:
+  aws:
+    s3:
+      bucket: umc-8th-dev
+      path:
+        review : review
+    region:
+      static: ap-northeast-2
+    stack:
+      auto:false
+    credentials:
+      accessKey: ${ AWS_ACCESS_KEY }
+      secretKey: ${ AWS_SECRET_KEY }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)